hcl2template/plugin.go

```diff
@@ -133,10 +133,22 @@ func (cfg *PackerConfig) initializeBlocks() hcl.Diagnostics {
 			// its body.
 			srcUsage := &(build.Sources[i])
 			if !cfg.parser.PluginConfig.Builders.Has(srcUsage.Type) {
+				detail := fmt.Sprintf(
```

Comment:
What do you think about reworking the message to read like the following?
```
The post-processor googlecompute-import is unknown by Packer, and is likely part of a plugin that is not installed.
You may find the needed plugin along with installation instructions documented on the Packer integrations page.

https://developer.hashicorp.com/packer/integrations?filter=googlecompute
```

Reply:
Sounds feasible to me, I'll reroll the PR ASAP